R/moma_arguments.R

```diff
@@ -320,24 +187,32 @@ cluster <- function(..., w = NULL, ADMM = FALSE,
         acc = acc, prox_eps = eps,
         P = "UNORDEREDFUSION"
     )
-    class(arglist) <- "moma_sparsity"
+    class(arglist) <- "_moma_sparsity_type"
     return(arglist)
 }
 
 
-#' Algorithm settings for solving a penalzied SVD problem
+#' Algorithm settings for solving the penalized SVD problem
+#'
+#' This function is used to specify the \code{pg_settings} argument
+#' in the \code{moma_*pca}, \code{moma_*cca}, and \code{moma_*ldaa}
+#' types of functions.
```

Comment:
Let's call this a "family" of functions - "type" has too many CS meanings already.

Reply:
Fixed.